R/deps.R

```diff
@@ -167,17 +167,7 @@ parse_one_remote <- function(x) {
       envir = asNamespace("devtools"), mode = "function", inherits = FALSE),
     error = function(e) stop("Unknown remote type: ", type, call. = FALSE))
 
-  simple_remote <- fun(repo)
```

Comment:
package2remote(remote_package_name(...)) looks at the *installed* package, which may have a different kind of remote than the requested remote.